jib-gradle-plugin/src/integration-test/java/com/google/cloud/tools/jib/gradle/SingleProjectIntegrationTest.java

```diff
@@ -97,32 +94,29 @@ private static void assertLayerSize(int expected, String imageReference)
   private static void assertDockerInspect(String imageReference)
       throws IOException, InterruptedException {
     String dockerInspect = new Command("docker", "inspect", imageReference).run();
-    MatcherAssert.assertThat(
-        dockerInspect,
-        CoreMatchers.containsString(
+    assertThat(dockerInspect)
+        .contains(
             "            \"Volumes\": {\n"
                 + "                \"/var/log\": {},\n"
                 + "                \"/var/log2\": {}\n"
-                + "            },"));
-    MatcherAssert.assertThat(
-        dockerInspect,
-        CoreMatchers.containsString(
+                + "            },");
```

Comment:
optional: could we use regex matching to only test the values that are relevant?

Reply:
I guess that involves a non-trivial amount of work, so maybe another time.